batteryInterface.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include <assert.h>
+#include <iostream>
+using namespace std;
+
+
+class BatteryElements
+{
+public:
+	static float current;
+	static float voltage;
+	static float temprature;
+};
+class BatterySpecification:public BatteryElements
+{
+public:
+	void BatterySpecificationPrinter();
+};
+class CurrentIndicator :public BatteryElements,public BatterySpecification
+{
+public:
+	static float currentMinThreshould;
+	static float currentMaxThreshould;
+	bool currentStatus();
+	
+};
+class VoltageIndicator :public CurrentIndicator
```

Comment:
What is the use of this inheritance? It is not being used in the program. Same question for the `TemperatureIndicator` as well.

Reply:
updated, now inheriting BatteryElements to get the actual values (Curr/volt/temp )and i created separate class considering single responsibility principal.